lightning/src/ln/channelmanager.rs

```diff
@@ -3471,11 +3473,11 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 		if msg.channel_id == [0; 32] {
 			for chan in self.list_channels() {
 				if chan.remote_network_id == *counterparty_node_id {
-					self.force_close_channel(&chan.channel_id);
+					let _ = self.force_close_channel(&msg.channel_id);
```

Comment:
Can you add a code comment here and below : "Untrusted messages from peer, we throw away the error if id points to a non-existent channel". A good warning to avoid vulns in in case of new work around this code path.

Reply:
Added in 821f6cd